mcmc/StatisticalUtils.h

```diff
@@ -504,10 +482,9 @@ inline void GetCredibleRegion(TH2D* const hist2D, const double coverage = 0.6827
 /// @param hist histograms from which we IQR
 inline double GetIQR(TH1D *Hist) {
 // *********************
-
   if(Hist->Integral() == 0) return 0.0;
 
-  double quartiles_x[3] = {0.25, 0.5, 0.75};
+  constexpr double quartiles_x[3] = {0.25, 0.5, 0.75};
```

Comment:
More a question for me, 
1. can this notbe a const vector instead of a pointer array?
2. why constexpr not const?

Reply:
const tells compiler that varibel will not change. This might help optmisation but mostly done for const correctnes. Also const may not be know at compile time

constexp tells this is constant and is known at compile time. Since it is knonw at compile time this results in SPEEEED. Not that we need for this code but why not